text/15_testing_basics.md

```diff
@@ -153,9 +153,9 @@ Consider a desktop email application.  Tasked with testing this, a black box tes
 
 Aspects of the system, such as what algorithm was used or what kind of memory allocation scheme is used, can be inferred by the black-box tester, but their concern is focused on the results of the system running.  For example, a black-box tester might notice that the system gets extremely slow when sorting a user's inbox when the user has thousands of messages.  The reason for this is a poor algorithm choice, but the tester is concerned with reporting the result that it is slow, not looking up or analyzing the algorithm.
 
-One could consider __white box testing__ to be the opposite of black box testing.  In white box testing, the tester has intimate knowledge of the codebase and tests code itself.  White box tests can test individual functions of the code, often looking at much more granular aspects of the system than black box tests.
```

Comment:
!!! I know !!! I just couldn't think of a better way to phrase it at the time.  This is great!